build/int.cloudbuild.yaml

```diff
@@ -14,7 +14,12 @@
 
 timeout: 3600s
 steps:
+- id: swap-module-refs
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && module-swapper']
```

Comment:
We can remove this step as we dont have any registry references to localize. We use this in some modules where we surface examples in cloud.google.com docs and we cannot use the relative `../..` [path in examples](https://github.com/amandakarina/terraform-google-cloud-run/blob/bd5c3517006574ca1a31047030a43ef01fd86201/examples/simple_cloud_run/main.tf#L18)
https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/infra/build/developer-tools/build/scripts/module-swapper for info incase we want to use this in the future.